CODEOWNERS

```diff
@@ -1 +1 @@
-* @Microsoft/ApplicationInsights-JS-owners
```

Comment:
This "should" have been picking up some people but it wasn't --- So this might still need a minor follup to remove the `@microsoft/` if this still isn't working -- documentation is unclear